conda-recipe/recipe.yaml

```diff
@@ -128,10 +128,12 @@ outputs:
       run:
         - python >=${{ python_min }}
         - ${{ pin_subpackage('napari-base', exact=True) }}
+        - aiohttp  # needed to open remote files by zarr
         - bermuda >=0.1.5
         - fsspec
         - napari-plugin-manager >=0.1.3,<0.2.0a0
         - numba >=0.57.1
+        - obstore  # needed to open remote files by zarr
```

Comment:
Currently, this isn't needed as fsspec is the default. But at some point this may become preferred?
Right now obstore is not part of the napari[all] installation.

Reply:
That said, we should have gotten this merged with or without obstore before 0.6.5 release I think. Now the bundle and napari[all] are out of sync.

Reply:
@jaimergp we missed this in the 0.6.5 release. If we merge this (with or without obstore) can we rebuild the 0.6.5 bundle?

Reply:
I'd still vote to remove, since obstore will not be used. If someone wants to use obstore to access S3, gcs, etc. they will need a plugin anyways.